common/djangoapps/third_party_auth/tests/utils.py

```diff
@@ -80,7 +85,7 @@ def _setup_provider_response_with_body(self, status, body):
 class ThirdPartyOAuthTestMixinFacebook(object):
     """Tests oauth with the Facebook backend"""
     BACKEND = "facebook"
-    USER_URL = FacebookOAuth2.USER_DATA_URL
+    USER_URL = FacebookOAuth2.USER_DATA_URL.format(version=API_VERSION)
```

Comment:
Right, did this in a jiffy, got excited that things fixed, and forgot about it. Will change it to match the settings.